pre-commit.sh

```diff
@@ -5,8 +5,10 @@
 # Script fails immediately when any command fails.
 set -e
 
+python -m pip install --user virtualenv
+
```

Comment:
`python` might refer to Python 2 for many people's systems -- could you keep the original virtualenv setup on this file?